nautobot/extras/tables.py

```diff
@@ -453,6 +456,13 @@ class JobResultTable(BaseTable):
         attrs={"td": {"class": "text-nowrap report-stats"}},
     )
 
+    def render_related_object(self, record):
+        """Custom rendering to be sure to only load related_object once."""
+        url = job_creator_link(value=None, record=record)
```

Comment:
Seems to me we could just fold the logic from `job_creator_link` into this method.

Reply:
Had to revert this because it was not working quite as intended. I did assert that the revised `job_creator_link()` you suggested coupled with the linkify in the `Column` does in fact only access `related_object` once per row.